### PR TITLE
SB-21589: Adding new notification service api

### PR DESCRIPTION
**Pull request by sknirmalkar89**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @sknirmalkar89,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://832.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://832.qa.docs.sunbird.org/4.2.0/apis/notificationserviceapi// <br /> - http://832.qa.docs.sunbird.org/4.2.0/apis/notificationserviceapi// <br />  <br/> **Modified sections are -** <br />

File: apis/notificationserviceapi/notificationserviceapi.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ paths:
       tags:
       - Notification APIs
       description: >-
-         - This API allows to send emai,sms,fcm and feed type notifications.
+         - This API allows to send emai,sms,device and feed type notifications. This will only be accessible internally by different service to send notification. Template configuration needs to be configured to use this service.
```
Comment (mohitga) [inline apis/notificationserviceapi/notificationserviceapi.yaml:23]: update emai to email

File: apis/notificationserviceapi/notificationserviceapi.yaml [hunk 2]
```diff
@@ -94,27 +96,21 @@ paths:
...
-                            name: Math's Activity
+                          
               c:
                summary: "Send SMS Notification"
                value:
                 request:
                   notifications:
                     - ids:
-                        - '12345'
+                        - '9824071234'
```
Comment (mohitga) [inline apis/notificationserviceapi/notificationserviceapi.yaml:106]: 9824071234 is not phone number?  we should not be leaking our phone number :)

File: apis/notificationserviceapi/notificationserviceapi.yaml [hunk 3]
```diff
@@ -0,0 +1,788 @@
...
+                  err: null
+                  status: null
+                  errmsg: "SERVER_ERROR"
+                 responseCode: SERVER_ERROR
+                 result: {}           
+  "/notification/v1/feed/read/{userId}":
+     get:
+      summary: Read User Notification Feed
+      description: >-
+        This API is used for getting in add notification of the user, it is based on the unique
```
Comment (mohitga) [inline apis/notificationserviceapi/notificationserviceapi.yaml:238]: change to: This API is used to get In-App notification for a userId, user Id uniquely identifies a user.

**Review by mohitga — COMMENTED** [review]
@sknirmalkar89 have a look at the comments and resolve them.

File: apis/notificationserviceapi/notificationserviceapi.yaml (hunk 2 above)
Comment (sknirmalkar89) [reply apis/notificationserviceapi/notificationserviceapi.yaml]: updated.

**Comment by amorphous-1** [comment]
Hello @sknirmalkar89,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://832.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://832.qa.docs.sunbird.org/4.2.0/apis/notificationserviceapi// <br /> - http://832.qa.docs.sunbird.org/4.2.0/apis/notificationserviceapi// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @sknirmalkar89,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://832.qa.docs.sunbird.org/4.2.0/ .<br/> **Note:** Only _4.2.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://832.qa.docs.sunbird.org/4.2.0/apis/notificationserviceapi// <br /> - http://832.qa.docs.sunbird.org/4.2.0/apis/notificationserviceapi// <br />  <br/> **Modified sections are -** <br />

**Pull request merged**